src/utilities/formInputConfigs.js

```diff
@@ -57,6 +62,16 @@ const FORM_INPUT_CONFIGS = seedObject => ({
     invalidMessage: formInputStrings.must_not_be_empty,
     isEditable: true,
   },
+  [FORM_INPUT_KEYS.SEARCH_FIRST_NAME]: {
+    type: 'text',
+    initialValue: '',
+    key: 'firstName',
+    validator: () => true,
+    isRequired: false,
```

Comment:
As searching, there is not requirement for a `firstName` to be specified. At a later stage may be worth adding some validation logic, but good chance of a refactor later down the line, so I think best to leave it simple for now!